websites/K/KQM/presence.ts

```diff
@@ -98,7 +98,7 @@ const browsingTimestamp = Math.floor(Date.now() / 1000),
 
 presence.on("UpdateData", async () => {
 	const presenceData: PresenceData = {
-			largeImageKey: "https://i.imgur.com/pDCUaZP.png",
+			largeImageKey: "https://uwu.so/adsHHUWVBSz",
```

Comment:
image must be 512x512 and be ending in .png/.jpg/etc.

Reply:
forgot